src/Navbar.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import {
+  FaBars,
+  FaEllipsisV,
+  FaMicrophone,
+  FaRegUserCircle,
+  FaSearch,
+} from 'react-icons/fa';
+
+const Navbar = () => {
+  const [focused, setFocused] = useState(false);
+
+  const handleFocus = () => {
+    setFocused(true);
+  };
+
+  const handleBlur = () => {
+    setFocused(false);
+  };
+
+  return (
+    <nav className="navbar">
+      <div className="navbar-center">
+        <div className="left">
+          <FaBars className="icon" />
+          <img src="https://www.freeiconspng.com/thumbs/youtube-logo-png/youtube-logo-png-transparent-image-5.png" alt="logo" className="logo" />
+        </div>
+
+        <div className="center">
+          <div className="search">
+            <form className={`${focused ? 'form has-focus' : 'form'}`}>
+              <div className="search-box-container">
+                <span className="icon-span">
+                  <FaSearch className="icon" />
+                </span>
+                <div className="search-box">
+                  <input type="text" className="input" placeholder="Search" onFocus={handleFocus} onBlur={handleBlur} />
+                </div>
```

Comment:
- [ ] Nice job @franqie , but your nav bar is currently not looking like that of youtube, kindly modify your styles :+1: 
youtube design :point_down: 
![image](https://user-images.githubusercontent.com/66097371/199101524-324a95b9-07ac-45a1-9e26-7d1593ae1277.png)
your design :point_down: 
![image](https://user-images.githubusercontent.com/66097371/199101606-ccca7cc0-98ea-4eaf-b51a-c6020bd7bb1e.png)

Reply:
Hi @VICTOR-CHUKWU,

Thank you for the review.
Kindly specify the style modifications I am to make. The nav bar I submitted was made to look like the pre-login screen of the one shown below.

Thanks.

![Annotation 2022-10-31 222550](https://user-images.githubusercontent.com/53271843/199190473-69bf26ca-6791-4d6e-8a77-3f99444aa357.png)

Reply:
hello @franqie can you make it look like the logged in screen? again the search icon in the input is light while yours is dark, in the logo `Youtube` your design is not yet matching the original design in fonts, also there is a superscript of country name `NG` above the logo